chingu-voyage4/Geckos-Team-32
# Pages/landing

## Purpose
* Create landing pages

## Approach/Fixes
* Code out landing page and post-landing page
* Add general styling to both
* Make pages responsive across all devices
* created a general page class `page-wrapper` for all future pages to properly align content

## Testing
* Download this PR to your local machine
* Check that the landing page shows a search bar
* Check that clicking the button renders the post-landing page

## Notes
* There is no `submit` button for the search bars yet (either the icon will act as the submit button later, or we'll have to create a separate button)
* Post-landing page has placeholder data for the time being
